Sources/TransferMethod/SelectTransferMethodTypePresenter.swift

```diff
@@ -66,12 +66,14 @@ final class SelectTransferMethodTypePresenter {
     /// Return the `TransferMethodTypeDetail` based on the index
     func getCellConfiguration(for index: Int) -> SelectTransferMethodTypeConfiguration {
         let transferMethodType = transferMethodTypes[index]
-        let feesProcessingTime = transferMethodType.formatFeesProcessingTime()
+        let feesProcessingTime = transferMethodType.code == "PAYPAL_ACCOUNT"
```

Comment:
This should not be changed. Ticket is to update only ListTransferMethodUI and not the Select

Reply:
reverted